packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -457,4 +457,33 @@ namespace FormData {
     @header contentType: "multipart/form-data",
     @multipartBody body: ComplexHttpPartsModelRequest,
   ): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+    Expect request:
+    ```
+    POST /upload HTTP/1.1
+    Content-Length: 428
+    Content-Type: multipart/form-data; boundary=abcde12345
+    
+    --abcde12345
+    Content-Disposition: form-data; name="temperature"
+    Content-Type: text/plain
+    
+    0.5
+    --abcde12345
+    ```
+    """)
+  @doc("Test content-type: multipart/form-data for non string")
+  @post
+  @route("/non-string")
+  op nonString(
```

Comment:
i wonder if it would be better to call it `float` instead of `nonString`, incase there are other non-string primitives we want to test. I'm ok either way though

Reply:
updated